### ISSUE TITLE
Incorrect Error message with multiple invalid names

### ISSUE THREAD

**Issue by PaulWestenthanner** [opwvhk/avro-schema-support#38]
**Description**
If an AVRO schema has multiple invalid field names (i.e. not complying with [A-Za-z0-9_]) the error message is incorrect. The invalid names are not highlighted.

**To Reproduce**

1. Create avro schema with 2 invalid field names, e.g.   ``` {
  "type": "record",
  "namespace": "my.namespace",
  "name": "myschema",
  "fields": [
    {
      "name": "/invalid1",
      "type": "string"
    },
    {
      "name": "/invalid2",
      "type": "string"
    }
  ]
} ```
 
3. Check warnings (either warnings tab or highlighted)
4. Invalid names won't be highlighted but the first line of the AVRO file suggesting `"type": "record"` is wrong

**Expected behavior**

All invalid names should be highlighted  
`"type": "record"` should not be highlighted

**Screenshots**
![image](https://user-images.githubusercontent.com/27921217/210750157-f3e36b3e-10d2-496e-b376-e32243e64a61.png)

![image](https://user-images.githubusercontent.com/27921217/210750213-edbcf7c8-2113-40b9-b8b6-38c498486d30.png)

**Versions:**
 - JetBrains Product: IntelliJ Ultimate

**Additional context**

I haven't checked if it is just with slashes or just with consecutive invalid fields.

**Comment by opwvhk** [comment]
Thank you for this well-written bug report! My compliments.

Reproducing this error is the easy part. Fixing it is not, unfortunately. The reason is that it's implemented using a JSON-schema. And although it's logically sound, that doesn't mean a validator is capable of correctly handling validation failures.

I've made some attempts, but I cannot yet find out how to adjust the JSON schema to improve the error message.

**Comment by PaulWestenthanner** [comment]
strange, for a single invalid field it works but I'm not familiar with the implementation
